textile/chat-features.textile

```diff
@@ -104,121 +104,146 @@ The status of any given Chat Room is the combination of the states of all of its
-
+As well as user-initiated operations, the room must monitor its underlying resources and act accordingly. This includes handling disconnections from the server, as well as discontinuities in channel messages.
+
+* @(CHA-RL4)@ This specification point has been removed. It was valid up until the single-channel migration.
+** @(CHA-RL4a)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL4a1)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL4a2)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL4a3)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL4a4)@ This specification point has been removed. It was valid up until the single-channel migration.
+** @(CHA-RL4b)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL4b1)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL4b2)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL4b3)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL4b4)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL4b5)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL4b6)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL4b7)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL4b8)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL4b9)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL4b10)@ This specification point has been removed. It was valid up until the single-channel migration.
+* @(CHA-RL11)@ The room must monitor the underlying realtime channel state, and update the room status accordingly.
+** @(CHA-RL11a)@ @[Testable]@ The room will subscribe to channel state changes on the underlying realtime channel.
+** @(CHA-RL11b)@ @[Testable]@ If a room lifecycle operation is in progress and a channel state change is received, the operation is no-op.
+** @(CHA-RL11c)@ @[Testable]@ If a room lifecycle operation is not in progress and a channel state change is received, the operation is no-op, the room status is set according to the underlying channel state, along with any associated error.
```

Comment:
```suggestion
** @(CHA-RL11c)@ @[Testable]@ If a room lifecycle operation is not in progress and such a channel state event is received, then the room status is set according to the underlying channel state, along with any associated error.
```

(clarifies that it's talking about the events from CHA-RL11, removes unintentional no-op mention)

Reply:
02a55eff